airflow/executors/local_executor.py

```diff
@@ -48,9 +49,13 @@ def run(self):
                 subprocess.check_call(command, shell=True)
                 state = State.SUCCESS
             except subprocess.CalledProcessError as e:
-                state = State.FAILED
-                self.logger.error("failed to execute task {}:".format(str(e)))
-                # raise e
+                if e.returncode == errno.EBUSY:
```

Comment:
The check is already there, e.g. a non 0 return code make the executor set itself to FAILED.